components/prism-tt2.js

```diff
@@ -0,0 +1,78 @@
+(function(Prism) {
+
+	var tt2_pattern = /\[%[\s\S]+?%\]|^%%*?/;
+
+	Prism.languages.tt2 = Prism.languages.extend('markup', {
```

Comment:
If you use the new markup-templating component (see my previous comment), you won't have to extend `markup` anymore.